sparse/src/KokkosSparse_Utils_cusparse.hpp

```diff
@@ -66,7 +66,8 @@ inline void cusparse_internal_safe_call(cusparseStatus_t cusparseStatus, const c
 
 // The macro below defines is the public interface for the safe cusparse calls.
 // The functions themselves are protected by impl namespace.
```

Comment:
I think the comment should be revised.
I could not find any documentation of the macro as part of the public interface. Do you know of anyone using it? Do you want to have the old one around but emit a deprecation warning instead of removing it?